docs/DeveloperGuide.adoc

```diff
@@ -279,104 +281,334 @@ Refer to the guide <<DevOps#, here>>.
+
+|`*` |admin |be able to toggle between different colourschemes for the
+app |
+
+|`*` |admin |archive appeal cases |view an uncluttered list of appeals
+
+|`*` |admin |get alerts on oversubscribed modules |receive early
+warnings of modules likely to receive appeal requests
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the System is the MAMS application and the
+Actor is the admin, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== UC01: Resolving an Appeal Request to Drop a Pre-allocated Module
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+Pre-condition: System has only just been opened, and is showing a list
+of appeals by default.
+
+1.  Admin views the details of the first appeal.
+2.  Admin displays the expanded information of the module.
+3.  Admin displays the expanded information of the student-of-interest.
+4.  Admin removes the pre-allocated module from student.
+5.  Admin marks the appeal as approved, with an approval message to be
+displayed to the student.
 +
-Use case ends.
+*Use case ends.*
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 2a. MAMS suggests an autocomplete based on the displayed appeal
+information *(Coming in v2.0)*
 +
-Use case ends.
+*Use case resumes at step 1.*
 
-* 3a. The given index is invalid.
+* 4a. Student workload goes below the minimum MC requirement for one
+semester
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 4a1. MAMS alerts the admin and asks for confirmation
+** 4a2. Admin confirms the decision.
 +
-Use case resumes at step 2.
+*Use case resumes at step 5.*
 
-_{More to be added}_
+[discrete]
+=== UC02: Approve second appeal in the list (a request from student to add a module)
 
-[appendix]
-== Non Functional Requirements
+*MSS*
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+Pre-condition: System has only just been opened, and is showing a list
+of appeals by default.
+
+1.  Admin views the details of the appeals in the displayed list
+2.  Admin requests to view appeal details of 2nd appeal in the list.
+3.  MAMS shows the appeal details(student particulars, module requested)
+4.  Admin requests to check potential clash of module requested by
+student with student’s existing modules
+5.  MAMS shows that there are no clashes
```

Comment:
Could add another step, that MAMS / system has to verify that the module capacity allows for another addition.